frontend/src/pages/index.jsx

```diff
@@ -13,20 +11,14 @@ import DashboardTransactions from "../components/dashboard/DashboardTransactions
 
 export default class extends React.Component {
   static async getInitialProps({ req }) {
-    const details = new DetailsApi(req).getDetails().catch(() => null);
     const blocks = new BlocksApi(req).getLatestBlocksInfo(8).catch(() => null);
-    const transactions = new TransactionsApi(req)
-      .getLatestTransactionsInfo(10)
-      .catch(() => null);
     return {
-      details: await details,
-      blocks: await blocks,
-      transactions: await transactions
```

Comment:
It is unfortunate that we give up on Server-side Rendering, but it seems to be a necessary compromise. :+1: